weave/panels_py/instructions/panel_trace_monitor.md

```diff
@@ -2,6 +2,66 @@
 
 To analyze Trace data, log `TraceSpanDict`s to a StreamTable. This can be achieved 3 different ways:
 
-- (Easy) Via 3rd party integrations (eg. Langchain)
+- (Easy) Via 3rd party integrations (eg. `Langchain`)
 - (Medium) Via decorating existing code with `@trace`
-- (Advanced) Via low-level construction of Spans.
+- (Advanced) Via low-level construction of Spans using `TraceSpanDict`.
+
+> Already using [W&B Tracer](https://docs.wandb.ai/guides/prompts)? Stay tuned: you soon will be able to load these traces in this dashboard!
+
+### Via Langchain Tracer
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/wandb/weave/blob/tim%2Fmake_tracer_a_featured_template/examples/prompts/trace_debugging/trace_quickstart_langchain.ipynb)
```

Comment:
minor todo? update these urls relative to main once merged?

Reply:
yup! exactly